apps/admin/frontend/src/lib/backends/backends.test.ts

```diff
@@ -32,113 +33,67 @@ function makeAdminBackend(): ElectionManagerStoreAdminBackend {
   const storage = new MemoryStorage();
   const logger = fakeLogger();
 
-  let nextElectionIndex = 1;
-  const db = new Map<
-    Id,
-    {
-      electionRecord: Admin.ElectionRecord;
-      memoryBackend: ElectionManagerStoreMemoryBackend;
-    }
-  >();
+  const electionId = 'test-election-1';
+  let memoryBackend: Optional<ElectionManagerStoreMemoryBackend>;
+  let electionRecord: Optional<Admin.ElectionRecord>;
 
   fetchMock
     .reset()
-    .get('/admin/elections', () => ({
-      body: typedAs<Admin.GetElectionsResponse>(
-        Array.from(db.values()).map(({ electionRecord }) => electionRecord)
-      ),
+    .get('/admin/elections/current', () => ({
+      body: typedAs<Admin.GetCurrentElectionResponse>({
+        status: 'ok',
+        electionRecord,
+      }),
     }))
     .post('/admin/elections', (url, request) => {
-      const electionId = `test-election-${nextElectionIndex}`;
-      nextElectionIndex += 1;
       const electionDefinition = safeParseJson(
         request.body as string,
         Admin.PostElectionRequestSchema
       ).unsafeUnwrap();
-      db.set(electionId, {
-        electionRecord: {
-          id: electionId,
-          electionDefinition,
-          createdAt: new Date().toISOString(),
-          isOfficialResults: false,
-        },
-        memoryBackend: new ElectionManagerStoreMemoryBackend({
-          electionId,
-          electionDefinition,
-        }),
+      electionRecord = {
+        id: electionId,
+        electionDefinition,
+        createdAt: new Date().toISOString(),
+        isOfficialResults: false,
+      };
+      memoryBackend = new ElectionManagerStoreMemoryBackend({
+        electionId,
+        electionDefinition,
       });
+
       return {
         body: typedAs<Admin.PostElectionResponse>({
           status: 'ok',
           id: electionId,
         }),
       };
     })
-    .patch('glob:/admin/elections/*', (url, request) => {
-      const match = url.match(/^\/admin\/elections\/(.+)$/);
-      const electionId = match?.[1] as Id;
-      const dbEntry = electionId && db.get(electionId);
-
-      if (!dbEntry) {
-        return { status: 404 };
-      }
-
-      const body = safeParseJson(
-        request.body as string,
-        Admin.PatchElectionRequestSchema
-      ).unsafeUnwrap();
-
-      if (typeof body.isOfficialResults === 'boolean') {
-        dbEntry.electionRecord = {
-          ...dbEntry.electionRecord,
-          isOfficialResults: body.isOfficialResults,
-        };
-      }
-
-      return { body: typedAs<Admin.PatchElectionResponse>({ status: 'ok' }) };
```

Comment:
Will have to do a lot of work to maintain this testing setup during a transition.